packages/cli/cli/src/cli.ts

```diff
@@ -992,6 +992,10 @@ function addDocsPreviewCommand(cli: Argv<GlobalCliOptions>, cliContext: CliConte
                     string: true,
                     hidden: true,
                     description: "Path of the local docs bundle to use"
+                })
+                .option("v2", {
+                    boolean: true,
+                    description: "Use openapi parser v2"
```

Comment:
Can we add this to the `api: ` block in `docs.yml` instead? That way someone can't docs dev v2 but then generate on v1 and encounter issues

Additionally if someone has 5 OpenAPIs, this gives you some upgrade flexibility